testing/scenario_app/lib/main.dart

```diff
@@ -10,40 +10,9 @@ import 'dart:isolate';
 import 'dart:typed_data';
 import 'dart:ui';
 
-import 'src/animated_color_square.dart';
-import 'src/platform_view.dart';
-import 'src/poppable_screen.dart';
-import 'src/scenario.dart';
-import 'src/send_text_focus_semantics.dart';
-import 'src/touches_scenario.dart';
-
-Map<String, Scenario> _scenarios = <String, Scenario>{
-  'animated_color_square': AnimatedColorSquareScenario(window),
-  'platform_view': PlatformViewScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
-  'platform_view_no_overlay_intersection': PlatformViewNoOverlayIntersectionScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
-  'platform_view_partial_intersection': PlatformViewPartialIntersectionScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
-  'platform_view_two_intersecting_overlays': PlatformViewTwoIntersectingOverlaysScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
-  'platform_view_one_overlay_two_intersecting_overlays': PlatformViewOneOverlayTwoIntersectingOverlaysScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
-  'platform_view_multiple_without_overlays': MultiPlatformViewWithoutOverlaysScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
-  'platform_view_max_overlays': PlatformViewMaxOverlaysScenario(window, 'Hello from Scenarios (Platform View)', id: 0),
-  'platform_view_cliprect': PlatformViewClipRectScenario(window, 'PlatformViewClipRect', id: 1),
-  'platform_view_cliprrect': PlatformViewClipRRectScenario(window, 'PlatformViewClipRRect', id: 2),
-  'platform_view_clippath': PlatformViewClipPathScenario(window, 'PlatformViewClipPath', id: 3),
-  'platform_view_transform': PlatformViewTransformScenario(window, 'PlatformViewTransform', id: 4),
-  'platform_view_opacity': PlatformViewOpacityScenario(window, 'PlatformViewOpacity', id: 5),
-  'platform_view_multiple': MultiPlatformViewScenario(window, firstId: 6, secondId: 7),
-  'platform_view_multiple_background_foreground': MultiPlatformViewBackgroundForegroundScenario(window, firstId: 8, secondId: 9),
-  'poppable_screen': PoppableScreenScenario(window),
-  'platform_view_rotate': PlatformViewScenario(window, 'Rotate Platform View', id: 10),
-  'platform_view_gesture_reject_eager': PlatformViewForTouchIOSScenario(window, 'platform view touch', id: 11, accept: false),
-  'platform_view_gesture_accept': PlatformViewForTouchIOSScenario(window, 'platform view touch', id: 11, accept: true),
-  'platform_view_gesture_reject_after_touches_ended': PlatformViewForTouchIOSScenario(window, 'platform view touch', id: 11, accept: false, rejectUntilTouchesEnded: true),
-  'tap_status_bar': TouchesScenario(window),
-  'text_semantics_focus': SendTextFocusScemantics(window),
-};
-
-Scenario _currentScenario = _scenarios['animated_color_square'];
+import 'src/scenarios.dart';
 
+// The scenarios are defined in `src/scenarios.dart`.
```

Comment:
nit: this comment may become outdated, and the language server should be able to get you where you need to go in an IDE.